### DISCUSSION TITLE
Set the colors of the sunburst chart?

### DISCUSSION THREAD

**Discussion by Oneto12** [ccbogel/QualCoder#964] [General]
Hi,

the interactive sunburst chart looks really cool and impressive. For a better fit to the semantic meaning of the codes, is there a way to change its color pattern?

Kind regards, Tim

**Comment by ccbogel** [comment]
I think it is possible to do some colour changes. But  I have not experimented with it yet.
https://plotly.com/python/sunburst-charts/

**Reply by Oneto12** [reply]
Thank you for the link. I just saw that plotly is also available for R (which I am more familiar with). I might try it out.

**Comment by ccbogel** [comment]
I have tried working on this, but found it too hard to implement
